### PR TITLE
Update IPR (Issue #626) and MCP to support configs with zero test harnesses

### PR DESCRIPTION
**Pull request by TaranFurnell**
and update MCP to use 1 arcsec

### CODE REVIEW

File: src/harness/testcases/WINNF_FT_S_MCP_testcase.py [hunk 1]
```diff
@@ -592,8 +596,8 @@ def performAggregateInterferenceCheck(self):
         if self.num_peer_sases > 0:
           ppa_ap_iap_ref_values = self.ppa_ap_iap_ref_values_list[index]
         else:
-          ppa_ap_iap_ref_values = [interference.dbToLinear(iap.THRESH_PPA_DBM_PER_RBW)] * len(ppa_aggr_interference)
-        # Compare the interference values calculated from both models.
+          ppa_ap_iap_ref_values = interference.dbToLinear(iap.THRESH_PPA_DBM_PER_RBW)
```
Comment (jni2000) [inline src/harness/testcases/WINNF_FT_S_MCP_testcase.py]: why are we removing multiplying "len(ppa_aggr_interference)" ? Also same question to the other places that the multiplying parts are removed.

File: src/harness/testcases/WINNF_FT_S_IPR_testcase.py [hunk 2]
```diff
@@ -210,37 +211,40 @@ def test_WINNF_FT_S_IPR_1(self, config_filename):
...
     # Heartbeat SAS UUT grants.
     n2_domain_proxy.heartbeatForAllActiveGrants()
     n3_domain_proxy.heartbeatForAllActiveGrants()
     # Get CbsdGrantInfo list of SAS UUT grants that are in an authorized state.
     grant_info = data.getAuthorizedGrantsFromDomainProxies([n2_domain_proxy, n3_domain_proxy])
 
-    # Check grants do not exceed each DPAs interference threshold.
-    for dpa, dpa_config in zip(dpas, config['dpas']):
+    # Initialize DPA objects and calculate movelist for each DPA.
+    dpas = []
```
Comment (jni2000) [inline src/harness/testcases/WINNF_FT_S_IPR_testcase.py]: Please can someone else help reviewing these DPA related addition ?

File: src/harness/testcases/WINNF_FT_S_MCP_testcase.py (hunk 1 above)
Comment (TaranFurnell) [reply src/harness/testcases/WINNF_FT_S_MCP_testcase.py]: When there are no peer SASs the interference to compare against is a single scalar threshold. There is no need to construct a list of the scalar, this is supported by the changes on lines 697-711 which compare directly against the number when it is a number.

File: src/harness/testcases/WINNF_FT_S_IPR_testcase.py (hunk 2 above)
Comment (sbdt-google) [reply src/harness/testcases/WINNF_FT_S_IPR_testcase.py]: In #615 , the DPA code has been updated to support None FAD objects. In these case, the checkInterference will check the SAS UUT active grants generated intereference against the absolute threshold.

File: src/harness/testcases/WINNF_FT_S_MCP_testcase.py (hunk 1 above)
Comment (jni2000) [reply src/harness/testcases/WINNF_FT_S_MCP_testcase.py]: ok, thanks.

**Comment by sbdt-google** [comment]
I will submit for Taran.

**Pull request merged**